src/main/java/net/snowflake/ingest/streaming/internal/RowBufferStats.java

```diff
@@ -16,6 +16,7 @@
 class RowBufferStats {
 
   private final int ordinal;
+  private final int fieldId;
```

Comment:
add comments on what's the difference between ordinal and fieldId, and what's expected values in them for (a) FDN column, (b) iceberg column primitive typed, (c) iceberg column non-primitive typed

Reply:
Added comments.